### PR TITLE
Add EIP check

### PR DESCRIPTION
**Pull request by mikolajsobolewski**

### CODE REVIEW

File: charts/wardend/Chart.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.3.1
+version: 2.4.1
```
Comment (jlehtimaki) [inline charts/wardend/Chart.yaml:19]:
```suggestion
version: 2.4.0
```

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

File: charts/wardend/values.yaml [hunk 2]
```diff
@@ -53,7 +53,8 @@ topologySpreadConstraints:
 
 eipController:
   enabled: false
-
+  check: false
+  retries: 20
```
[the remark this answers is not in the document]
Comment (mikolajsobolewski) [reply charts/wardend/values.yaml:57]: I think this might be useful in case of problems with EIP

[the remark this answers is not in the document]
Comment (jlehtimaki) [reply charts/wardend/values.yaml:57]: I think it's better that the container does not even start without EIP and like I said, we can use this with the default kubernetes behaviour:
https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#detailed-behavior

[the remark this answers is not in the document]
Comment (mikolajsobolewski) [reply charts/wardend/values.yaml]: I'd removed retries.

File: charts/wardend/scripts/init.sh [hunk 3]
```diff
@@ -35,3 +48,21 @@ if [ -e "$SRC_NOKE_KEY" ]; then
       echo "Node Key is up to date. No action needed."
   fi
 fi
+
+if [ -n "$EIP_CHECK" ]; then
+    if [ -z "$EIP_RETRIES" ]; then
```
Comment (jlehtimaki) [inline charts/wardend/scripts/init.sh:53]: Need to remove the EIP_RETRIES from the script as well

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

File: charts/wardend/scripts/init.sh (hunk 3 above)
Comment (mikolajsobolewski) [reply charts/wardend/scripts/init.sh]: Oops, sorry, forgot the duplicate

**Review by jlehtimaki — APPROVED** [review]

**Pull request merged**